pipeline/splicing/calcVarPriors.py

```diff
@@ -670,20 +688,32 @@ def getRefAltSeqs(variant, rangeStart, rangeStop):
     varChrom = getVarChrom(variant)
     varStrand = getVarStrand(variant)
```

Comment:
i see strand as a variable elsewhere (e.g. getAltSeq right above this function). If they refer to the same thing it will be easier to read if it's always the same variable name

Reply:
Gotcha, changed them all to `varStrand` for consistency